native/cocos/renderer/gfx-vulkan/VKCommands.cpp

```diff
@@ -958,7 +959,7 @@ void cmdFuncCCVKCreateComputePipelineState(CCVKDevice *device, CCVKGPUPipelineSt
 
     ///////////////////// Creation /////////////////////
 
-    VK_CHECK(vkCreateComputePipelines(device->gpuDevice()->vkDevice, device->gpuDevice()->vkPipelineCache,
+    VK_CHECK(vkCreateComputePipelines(device->gpuDevice()->vkDevice, device->pipelineCache()->getHandle(),
```

Comment:
It seems `device->pipelineCache()` might return null?

Reply:
oh, only gles3 might return null

Reply:
Might add some asserts to make sure precondition is met.
It is useful when refactoring code.

Reply:
Got it, I will add some asserts.